mobile/lib/l10n/intl_nl.arb

```diff
@@ -620,7 +620,7 @@
   "yesCancel": "Ja, opzeggen",
   "failedToRenew": "Verlengen mislukt",
   "failedToCancel": "Opzeggen mislukt",
-  "twoMonthsFreeOnYearlyPlans": "Krijg 2 maanden gratis op jaarlijkse abonnementen",
+  "twoMonthsFreeOnYearlyPlans": "Krijg 2 maanden gratis bij jaarlijkse abonnementen",
```

Comment:
The original text is
> 2 months free on yearly plans.

As per online translator, the current dutch to en translation is

> Get 2 months free on annual subscriptions

And update one is

> Get 2 months free with annual subscriptions

(Obviously, i don't know Dutch, was just curious what was getting fixed).

Should we also update the english translation?

For directly updating on crowdin, we can use following links
- [mob](https://crowdin.com/editor/ente-photos-app/11/en-nl?view=comfortable&filter=basic&value=5#q=twoMonthsFreeOnYearlyPlans)

- [web](https://crowdin.com/editor/ente-photos-web/9/enus-nl?view=comfortable&filter=basic&value=5#q=two_months_free)

Reply:
The wrong word got used in that context "Op" would be literally on top of something while "bij" would imply extra